.github/workflows/build.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: setup node
       uses: actions/setup-node@v4
       with:
-        node-version: '21.x'
```

Comment:
(Noticed the `.x` does not resolve https://github.com/jaymzh/main-branch-check-action/actions/runs/9169005598/job/25208725652#step:3:8)